cardano-cli/src/Cardano/CLI/EraIndependent/Address/Run.hs

```diff
@@ -38,6 +41,9 @@ import Data.ByteString.Char8 qualified as BS
 import Data.Function
 import Data.Text.IO qualified as Text
 
+import Vary (Vary)
+import Vary qualified
```

Comment:
Could you add grouping of this import with other hackage packages in `fourmolu.yaml`? A standalone import group looks odd.

Btw. I think author should've chosen more conventional module name like `Data.Vary` or something.